Makefile

```diff
@@ -11,7 +11,7 @@
 #   Stable releases:   "1.0.0"
 #   Pre-releases:      "1.0.0-alpha.1", "1.0.0-beta.2", "1.0.0-rc.3"
 #   Master/dev branch: "1.0.0-dev"
-VERSION=15.0.0-dev
+VERSION=15.0.0-dev.isaiah.1
```

Comment:
Ensure this change does not get merged please.

What I usually do is create a `camh/test/foo` branch for `camh/foo` changes and run the test build from there - it keeps the PR branch clean and you just switch back to it and delete the test branch to clean up. The tags still hang around though. We should consider cleaning up dev tags every now and then.

Reply:
Good call, I'll do that in the future. Should be polished for review now.